python/ccxt/base/exchange.py

```diff
@@ -466,6 +466,13 @@ def __init__(self, config={}):
             self.session.trust_env = self.requests_trust_env
         self.logger = self.logger if self.logger else logging.getLogger(__name__)
 
+        type_names = {
+            'string': str,
+            'number': float,
+        }
+        assigned_type = self.safe_string_lower(self.options, 'number', 'number')
+        self.number = type_names[assigned_type]
```

Comment:
wouldnt this be a breaking change for current users, who don't use this new approach, and just instantiate with `okx({number: String})` directly?

maybe the better way could be to check, if string-format value was assigned to `.number` prop, convert that to type instead of string value.
so, i.e. keys.json:
```
"alpaca": {
    "number": "string"
}
```

and detect&convert "string" into type in base classes.

Reply:
@ttodua What are you trying to say? The point is to be able to do this

```
"alpaca": {
    "number": "string"
}
```

There's no breaking change here

Reply:
what I am trying to say is, what about ppl who has userland codes like:
```
ex = new ccxt.binance ({number: String});
```
their assignment to String will be overwritten to `number`, because of how this PR adds code into base constructor